libiocage/lib/helpers.py

```diff
@@ -149,17 +143,25 @@ def validate_name(name: str) -> bool:
     return bool(_validate_name.fullmatch(name))
 
 
-def parse_none(data):
+def parse_none(data) -> str:
     if data is None:
         return None
```

Comment:
does this (return value of `str`) mean a `return None` will be auto-converted to `"None"` (`str`)?